src/app/account/change-pwd/change-pwd.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from '@angular/core';
+import {User} from '../../model/user';
+import {ActivatedRoute, Router} from '@angular/router';
+import {UserService} from '../../services-api/user.service';
+import {MessageService} from 'primeng/api';
+
+@Component({
+  selector: 'app-change-pwd',
+  templateUrl: './change-pwd.component.html',
+  styleUrls: ['./change-pwd.component.css']
+})
+export class ChangePwdComponent implements OnInit {
+  model: User = new User();
+  confirmPassword: string
+  loading = false;
+
+  constructor(
+    private route: ActivatedRoute,
+    private router: Router,
+    private userService: UserService,
+    private messageService: MessageService) { }
+
+  ngOnInit() {
+  }
+
+  changePwd(){
+    this.loading = true;
+    if (this.model.password === this.confirmPassword)
+    {
```

Comment:
Peux-tu mettre l'accolade sur la même ligne que le if ?
```suggestion
    if (this.model.password === this.confirmPassword) {
```